sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/OptimizeJsonExprs.scala

```diff
@@ -82,12 +82,12 @@ object OptimizeJsonExprs extends Rule[LogicalPlan] {
         child
 
       case g @ GetStructField(j @ JsonToStructs(schema: StructType, _, _, _), ordinal, _)
-          if schema.length > 1 =>
+          if schema.length > 1 && j.options.isEmpty =>
```

Comment:
Ur, you should not use `[TEST]` tag when you touch `src/main`.

Reply:
Removed.